DP-22331: flexible organizations

tom-fleming opened the pull request:
**This branch has a corresponding Mayflower integration branch: https://github.com/massgov/mayflower/pull/1465**

arthurbaghdas committed:
DP-22330: Setup org content type (#832)

* DP-22153: Enable debugging for local development (#818)

* DP-22153: Enable debugging for local development

* CHangelog

* PHPCS issue fix

* PHPCS issue fix

* PHPCS issue fix

* Adding settings.local.yml file to the repo

* DP-22083 Upgrade Mandrill to D9 compatible version, remove patch. (#819)

* DP-22194 - Updated People view to include count of revisions and relative date filter (#816)

* Added views aggregator plus module and set revision count fields and sort - DP-22194

* Added changelog - DP-22194

* changed placeholder text

Co-authored-by: Joe Galluccio <joseph.galluccio@state.ma.us>
Co-authored-by: abaghdasaryan <arthur.baghdasar@gmail.com>

* DP-20892 Responsive iframe height (#605)

* Added and configured form display of iframe height config field - DP-20892

* Made height field conditional - DP-20892

* Removed height config from paragraph display - DP-20892

* Added iframe resize scripts - DP-20892

* Modified iframe paragraph to assign non-numeric height value to iframe when set to full - DP-20892

* Added changelog - DP-20892

* Cleaned up resizer scripts to fix eslint errors - DP-20892

* Revert "Cleaned up resizer scripts to fix eslint errors - DP-20892"

This reverts commit 5532666fa393c852df1e24ab2b9eae2a74b999af.

* Revert "Revert "Cleaned up resizer scripts to fix eslint errors - DP-20892""

This reverts commit 3ec20f571d46e4c19882e9ec6865193272376210.

* Pushing up more cleaned up version for testing

* Update docroot/themes/custom/mass_theme/overrides/js/iframe_resizer_iframe.js

Co-authored-by: Minghua Sun <clairesunstudio@gmail.com>

* Update docroot/themes/custom/mass_theme/overrides/js/iframe_resizer_parent.js

Co-authored-by: Minghua Sun <clairesunstudio@gmail.com>

* Fixed Twig template - DP-20892

* Updated help text - DP-20892

* Cleaned up parent script - DP-20892

* Fixed script link - DP-20892

* Fixed conditional field logic for existing content - DP-20892

* Moved help text to 100% field - DP-20892

* Added 100% iframe confirmation field to show conditionally - DP-20892

* Updated conditional fields JS, moved setting full height value to preprocess - DP-20892

* Added library - DP-20892

* Addressed some issues affecting paragraph value output

* Fixed setting auto height and library - DP-20892

* Additional adjustments to iframe display and 100% confirmation - DP-20892

* Fixed height setting for iframes - DP-20892

* Added simplified, more es6-like version of resizer - DP-20892

* Added simplified, more es6-like version of resizer - DP-20892

* Modified script to theoretically work with multiple iframes - DP-20892

* Added a little extra padding to prevent scrollbar - DP-20892

* Update iframe_resizer_parent.js

* Update iframe_resizer_iframe.js

* Update iframe_resizer_iframe.js

* lint fixes

* more eslint fix

* fix code

* Update conf/drupal/config/field.storage.paragraph.field_iframe_height_config.yml

Co-authored-by: Minghua Sun <clairesunstudio@gmail.com>

* Fixed script in help text - DP-20892

* word wrap radio form wrappers in the admin theme - DP-20892

* Update docroot/themes/custom/mass_theme/overrides/js/iframe_resizer_iframe.js

* Update conf/drupal/config/field.storage.paragraph.field_iframe_height_config.yml

* Update docroot/themes/custom/mass_theme/overrides/js/iframe_resizer_iframe.js

* Changelog change - DP-20892

* Removed changelog - DP-20892

* Re-added changelog - DP-20892

* renamed changelog - DP-20892

* Removed / from changelog - DP-20892

* Switched changelog to use related issue - DP-20892

* Modified changelog description - DP-20892

* Removed changelog - DP-20892

* Added new changelog - DP-20892

* Added back other changelog - DP-20892

* DP-20892 - Update iframe to handle more events

* DP-20892_iframe-height

* Eslint fi

Co-authored-by: Minghua Sun <clairesunstudio@gmail.com>
Co-authored-by: Diego Diestra <ddiestras@gmail.com>

* DP-22294: Change Data topic field widget (#822)

* DP-22294: Change Data topic field widget

* Changelog

* DP-22249 Office Hours Module upgrade and theming (#814)

* DP-22249 Office Hours Module upgrade and theming

* Changelog

* DP-22249 Updating lock file

* Mayflower/dp dp 22375 mayflower version 11.7.1 (#826)

* update mayflower artifacts

* Updated composer.lock - DP-22375

* add changelog

Co-authored-by: Tom Fleming <tom@lastcallmedia.com>

* Dp 9770 signee linked logo test (#824)

* set up ariaHidden for promo image link

* add changelog for DP-9770

* reposition ariaHidden

* pull in mayflower branch

* remove test comment

* remove mayflower branch for test

* set up ariaHidden in preprocess

* fix validation error

* change Mayflower version to mayflower/DP-22122_image-promo-press for test

* revert composer.lock changes to match develop

Co-authored-by: Minghua Sun <clairesunstudio@gmail.com>

* Pre changes

* DP-22255: Allow authors to choose the type of ordered list style they need for a list (#820)

* DP-22255: Allow authors to choose the type of ordered list style they need for a list

* CHangelog

* Behat test fix

* Patching lib

* COmposer.lock update

Co-authored-by: Minghua Sun <clairesunstudio@gmail.com>

* changelog update and remove old changelog files

* Add validation

* Changelog

* DP-22277: Don't send authors to media page in Drupal authoring (#821)

* DP-22277: Don't send authors to media page in Drupal authoring

* Changelog

* PHPCS

Co-authored-by: Nathan Andersen <nathan.andersen@codeandtheory.com>
Co-authored-by: Tom Fleming <tom@lastcallmedia.com>
Co-authored-by: Joe Galluccio <joseph.galluccio@state.ma.us>
Co-authored-by: Minghua Sun <clairesunstudio@gmail.com>
Co-authored-by: Diego Diestra <ddiestras@gmail.com>
Co-authored-by: Matt Kleve <matt.kleve@gmail.com>
Co-authored-by: Yukiko Gannett <yukikogannett@gmail.com>
Co-authored-by: CircleCI Deployment Bot <circleci@example.com>
Co-authored-by: Moshe Weitzman <weitzman@tejasa.com>

Javi-er committed:
DP-22483 - Adding featured topics paragraph and theming it.

Javi-er committed:
Merge branch 'develop' into DP-22331_flex-orgs

arthurbaghdas committed:
DP-22538: Make paragraph for Board Members on org page (#856)

* Board members section

* Changelog

* Changelog

* Remove debug code

weekbeforenext committed:
Merge branch 'develop' into DP-22331_flex-orgs

Commit:
DP-22403: Make existing paragraph types on org page available for sections (#843)

* DP-22403: Adding the Featured Item Mosaic, Featured Message, and Organization Grid paragraphs to the Organization Section paragraph type, Content field. Also hiding heading fields in Featured Item Mosaic and Featured message paragraphs.

* DP-22403: Adding the changelog file

* DP-22403: Removing duplicate headings from Featured Item Mosaic and Organization Grid theme templates

* DP-22403: Adding logic to the Organization Section template to allow for an optional section heading

* DP-22403: Enabling the Featured Message heading field

Commit:
DP-22534: Make paragraph for News on org page (#850)

* DP-22534: Adding an Organization News paragraph, making it addable in the Organization Sections paragraph, and refactoring related logic to apply to the new paragraph

* DP-22534: Adding changelog file

* DP-22534: Updating the mass_theme_preprocess_paragraph__org_news() description

Commit:
DP-22535: Make paragraph for Events on org page (#853)

* DP-22535: Adding an Organization Events paragraph and refactored logic to display on an org_page node

* DP-22535: Fixing some coding standards errors

Commit:
DP-22536:  Make paragraph for Locations on org page (#860)

* DP-22536: Converting map fields from the Organization Page content type to an Organization Locations paragraph

* DP-22536: Adding the changelog file

* DP-22536: Fixing PHP coding standards errors

weekbeforenext committed:
DP-22331: Fixing field configuration after merge conflicts resolved

Commit:
#DP-22483  adding headline

Commit:
DP-22483 adjusted field.

Commit:
DP-22533: Allow authors to hide short description on org page (#867)

* DP-22533: Adding a checkbox to the org_page content type to hide the short description on the page display

* DP-22533: Adding the changelog file

Commit:
DP-22537: Make paragraph for Related Organizations on org page (#870)

* DP-22537: Adding an Organization Related Organizations paragraph type and updating the Organization page content type to use it

* DP-22537: Adding changelog file

Commit:
Removing heading from featured topics.

Commit:
DP-22403: Make existing paragraph types on org page available for sections (follow-up) (#874)

* DP-22403: Adding the Featured Item Mosaic, Featured Message, and Organization Grid paragraphs to the Organization Section paragraph type, Content field. Also hiding heading fields in Featured Item Mosaic and Featured message paragraphs.

* DP-22403: Adding the changelog file

* DP-22403: Removing duplicate headings from Featured Item Mosaic and Organization Grid theme templates

* DP-22403: Adding logic to the Organization Section template to allow for an optional section heading

* DP-22403: Enabling the Featured Message heading field

* DP-22403: Reinstating the maxlength settings for field_featured_message_heading and moving the field validation for field_featured_item_mosaic_items to the nested paragraph

weekbeforenext committed:
Merge branch 'develop' into DP-22331_flex-orgs

Commit:
DP-22644: Org page migration scaffold (#877)

* DP-22644: Adding a deploy function scaffolding for migrating org_page content to organization sections

* DP-22644: Adding changelog file

* DP-22644: Fixing coding standards errors

Commit:
DP-22654: Migrate org page Featured Message to sections (#880)

* DP-22654: Adding logic to migrate Featured Message content for org pages

* DP-22654: Adding the changelog file

* DP-22654: Fixing a PHP standards error

weekbeforenext committed:
Merge branch 'develop' into DP-22331_flex-orgs

Commit:
DP-22655: Migrate org page Featured Items Mosaic to sections (#882)

* DP-22391: Remove old alerts blocks (#865)

* DP-22391: Removing the mayflower AjaxPatternBlock instance configuration, plugin, and related JS

* DP-22391: Adding changelog file

* DP-22655: Adding the logic to migrate data for org_pages featured items data

Commit:
DP-22656: Adding logic to migrate Our Organizations data to Organization Section paragraphs for org_pages (#885)

Commit:
DP-22659: Adding logic to migrate News data to Organization Section paragraphs for org_pages (#886)

arthurbaghdas committed:
DP-22658: Migrate org page Featured Topics to sections (#890)

* DP-22658: Migrate org page Featured Topics to sections

* DP-22658: Migrate org page Featured Topics to sections

arthurbaghdas committed:
DP-22633: Board members migration (#884)

* DP-22633: Migrate org page Board Members to sections

* DP-22633: Migrate org page Board Members to sections

weekbeforenext commented on changelogs/DP-22656.yml:
```diff
@@ -37,5 +37,5 @@
 #    issue: DP-19843
 #
 Changed:
-  - description: Migrated Our Organizations data to sections for Organization pages.
```
@arthurbaghdas I think this changelog is suppose to be for DP-22658 instead of DP-22656. It overwrote another changelog file. Can you fix it?

arthurbaghdas replied:
Thanks for noticing that. Fixed now

weekbeforenext commented on docroot/modules/custom/mass_content/includes/mass_content.org_migration.inc:
```diff
@@ -130,7 +130,20 @@ function _mass_content_org_page_migration_what_would_you_like_to_do(&$node) {
+    $pid = $paragraph->id();
+    $pid_revision = $paragraph->getRevisionId();
+    $node->set("field_ref_card_view_6", []);
+    $node->field_organization_sections[] = [
```
I created the reusable function _mass_content_org_page_migration_add_section() at the top of this include file to consistently add new sections to the org_page node, if you want to use that here.

weekbeforenext replied:
We also need to create a new organization section paragraph instead of moving the paragraph reference.

weekbeforenext commented on docroot/modules/custom/mass_content/includes/mass_content.org_migration.inc:
```diff
@@ -189,7 +189,15 @@ function _mass_content_org_page_migration_news(&$node) {
+  if (!empty($node->field_boards->target_id)) {
+    $pid = $node->field_boards->target_id;
+    $pid_revision = $node->field_boards->target_revision_id;
+    $node->set("field_boards", []);
```
The field_boards paragraph needs to be referenced within a new organization section, right?

arthurbaghdas replied:
Yes

Commit:
DP-22654: Migrate org page Featured Message to sections (follow-up) (#894)

* DP-22654: Adding logic to migrate Featured Message content for org pages

* DP-22654: Adding the changelog file

* DP-22654: Fixing a PHP standards error

* DP-22654: Adding logic to remove old field data and hide the related fields

Commit:
DP-22655: Migrate org page Featured Items Mosaic to sections (follow-up) (#895)

* DP-22655: Adding the logic to migrate data for org_pages featured items data

* DP-22655: Adding logic to remove old field data and hide the related fields

Commit:
DP-22656: Migrate org page Organization Grid to sections (follow up) (#897)

* DP-22656: Adding logic to migrate Our Organizations data to Organization Section paragraphs for org_pages

* DP-22656: Adding logic to remove old field data and hide the related fields

Commit:
DP-22659: Migrate org page News to sections (follow up) (#898)

* DP-22659: Adding logic to migrate News data to Organization Section paragraphs for org_pages

* DP-22659: Adding logic to remove old field data and hide the related fields

arthurbaghdas committed:
Changelog fix

arthurbaghdas committed:
Refactored the code for featured topics migration

arthurbaghdas committed:
Refactored the code for board members migration

Commit:
DP-22537: Make paragraph for Related Organizations on org page (follow up) (#902)

* DP-22537: Adding an Organization Related Organizations paragraph type and updating the Organization page content type to use it

* DP-22537: Adding changelog file

* DP-22537: Changing the paragraph label from Organization Related Organizations to Related Organizations

* DP-22537: Reverting config file to match integration branch

weekbeforenext committed:
Merge branch 'develop' into DP-22331_flex-orgs

Commit:
DP-22483 adjusting logic.

Commit:
DP-22660: Migrate org page Events to sections (#912)

* DP-22660: Adding logic to migrate Organization events field data to Organization Sections

* DP-22660: Hiding the Event quantity field on Organization node form

weekbeforenext committed:
DP-22331: Adding the mayflower integration branch to composer

Commit:
DP-22689: Make About section for org page (#906)

* DP-22689: Adding About paragraph to Organization Section content field and removing the title from the about section template

* DP-22689: Setting the mayflower-artifacts package to the feature branch with changes

* DP-22689: Adjusting logic for the SocialLinks computed field to display links from Organization Section paragraph in About section

* DP-22689: Reverting code refactor that is not necessary

weekbeforenext committed:
Merge branch 'develop' into DP-22331_flex-orgs

Commit:
DP-22689: Make About section for org page (followup) (#917)

* DP-22689: Adding About paragraph to Organization Section content field and removing the title from the about section template

* DP-22689: Setting the mayflower-artifacts package to the feature branch with changes

* DP-22689: Adjusting logic for the SocialLinks computed field to display links from Organization Section paragraph in About section

* DP-22689: Reverting code refactor that is not necessary

* DP-22689: Hiding the About field and field group on org_page node form

weekbeforenext committed:
Merge branch 'develop' into DP-22331_flex-orgs

Commit:
DP-22661: Migrate org page Locations to sections (#918)

* DP-22661: Adding logic to migrate location data for Org pages to Org sections and hiding the fields on the org_page form

* DP-22661: Adding the changelog file

Commit:
DP-22689: Make About section for org page (second follow up) (#920)

* DP-22689: Adding About paragraph to Organization Section content field and removing the title from the about section template

* DP-22689: Setting the mayflower-artifacts package to the feature branch with changes

* DP-22689: Adjusting logic for the SocialLinks computed field to display links from Organization Section paragraph in About section

* DP-22689: Reverting code refactor that is not necessary

* DP-22689: Hiding the About field and field group on org_page node form

* DP-22689: Updating the About paragraph type label

weekbeforenext committed:
Merge branch 'develop' into DP-22331_flex-orgs

Javi-er committed:
Dp 22561 wwyltd paragraph (#900)

* DP-22561 What would you like to do paragraph

* DP-22483 final adjustments

* Create DP-22561.yml

* Adjusting WWYLTD paragraph, removing non necessary fields and re formatting to adapt to MF component

* DP-22561: Deleting old changelog files

Co-authored-by: Javier Reartes <jreartes@taoti.com>
Co-authored-by: April Sides <weekbeforenext@gmail.com>

Javi-er committed:
DP 22652 Make social media section for org page (#916)

* DP-22652 #comment added new social links paragraph for organizations.

* Linting

* DP-22652: Fixing Organization Section Content field configuration

Co-authored-by: Javier Reartes <jreartes@taoti.com>
Co-authored-by: April Sides <weekbeforenext@gmail.com>

Commit:
DP-22661: Migrate org page Locations to sections (followup) (#921)

* DP-22661: Adding logic to migrate location data for Org pages to Org sections and hiding the fields on the org_page form

* DP-22661: Adding the changelog file

* DP-22661: Refactoring Organization Locations migration and locations page

* DP-22661: Updating locations page logic to find a paragraph using the right revision key

* DP-22661: Fixing logic for showing org_page locations

* DP-22661: Fixing PHP coding standards error

Javi-er committed:
DP-22666 added contact us paragraph (#922)

Co-authored-by: Javier Reartes <jreartes@taoti.com>
Co-authored-by: April Sides <weekbeforenext@gmail.com>

Commit:
DP-22661: Fixing the org locations page logic (#923)

Commit:
DP-22483 cleaning up.

Commit:
Merge branch 'DP-22331_flex-orgs' of github.com:massgov/openmass into DP-22331_flex-orgs

Commit:
DP-22690: Adding logic to migrate About section to organization sections (#927)

Commit:
DP-22685: Migrate org page "Who we serve" to sections (#931)

* DP-22685: Adding logic to migrate the 'Who we serve' data to Organization sections and fixing an issue with the About field migration

* DP-22685: Updating the org_page form display to hide the body field

Commit:
DP-22685: Fixing a typo in the 'Who we serve' migration header (#932)

Javi-er committed:
Migrate related organizations (#936)

* Create DP-22662.yml

* Related orgs migration function.

* hiding the related orgs field from form view.

Co-authored-by: Javier Reartes <jreartes@taoti.com>

weekbeforenext committed:
Merge branch 'develop' into DP-22331_flex-orgs

Commit:
DP-22746: Address Behat errors in flexible org integration branch (#941)

* DP-22746: Fixing org_page form display and Behat tests

* DP-22746: Updating org_page Behat tests

* DP-22746: Removing Behat test lines for events on org_page nodes

Javi-er committed:
#DP-22657 migrating What would you like to do content. (#935)

* #DP-22657 migrating What would you like to do content.

* tidying.

* removed services field from view

* Adjusting migration

* removed more services from form view

* Linting code.

Co-authored-by: Javier Reartes <jreartes@taoti.com>
Co-authored-by: April Sides <weekbeforenext@gmail.com>

Commit:
DP-22746: Address Behat errors in flexible org integration branch (followup) (#946)

* DP-22746: Fixing org_page form display and Behat tests

* DP-22746: Updating org_page Behat tests

* DP-22746: Removing Behat test lines for events on org_page nodes

* DP-22746: Removing Behat test lines for fields that will be removed from org_pages

clairesunstudio committed:
update mayflower for compact topic cards

weekbeforenext committed:
Merge branch 'develop' into DP-22331_flex-orgs

Commit:
DP-22690: Migrate org page About to sections (followup) (#953)

* DP-22690: Hiding short name field on org_page, moving Get updates links field to the More tab, and removing the About/Details tab

* DP-22690: Removing short name field from org_page Behat tests

Javi-er committed:
DP-22666 Adjusting "Contact and Logo" to use existing fields instead of new fields. (#951)

* Adjusted to use existing fields instead of new fields.

* updating description

Co-authored-by: Javier Reartes <jreartes@taoti.com>

Javi-er committed:
Feature/dp 22688 contact logo migration (#930)

* #DP-22688 migrated data to the new paragraph (contact and logo)

* changelog

* Adjusting migration for just placing the block instead. Data will be retrieved from the original fields.

Co-authored-by: Javier Reartes <jreartes@taoti.com>
Co-authored-by: April Sides <weekbeforenext@gmail.com>

Javi-er committed:
DP-22688 adding theme preprocess variables to template (#974)

Javi-er committed:
DP-22666 adding notice to contact logo paragraph. (#982)

weekbeforenext committed:
Merge branch 'develop' into DP-22331_flex-orgs

Javi-er committed:
DP-22657 Sending the number of items to the component. (#983)

weekbeforenext committed:
DP-22331: Updating the mayflower integration branch in composer

weekbeforenext committed:
Merge branch 'develop' into DP-22331_flex-orgs

Commit:
DP-22947: Removing the Additional Resources field from the Organization Section paragraph, editing the Heading field help text, and tweaking the Organization content type form display (#993)

Javi-er committed:
Changing  "content" with "contact" (#995)

weekbeforenext committed:
DP-22331: Updating the mayflower-artifacts package

Commit:
DP-22747: Run Backstop on flexible org branch (#994)

* DP-22747: Reinstating the org_page postContent template logic and removing unnecessary logic from org_page preprocess theme function

* DP-22747: Reinstating the pageBanner logic in the org_page preprocess function

* DP-22747: Removing the org /services page

* DP-22747: Fixing the org_page schema disambiguatingDescription value

* DP-22747: Updating org_page event migration logic to skip migration for nodes that have no past or upcoming events

Commit:
DP-22958: Adding a checkbox field to the Organization Section paragraph that hides the heading field when enabled and updating the org_page migrations to include a heading for all sections (#996)

Javi-er committed:
DP-22961 Org section separator changes for feature branch (#1003)

* DP-22961 adding separators.

* adding separator field to paragraph.

* DP-22961: Adding changelog file

Co-authored-by: April Sides <weekbeforenext@gmail.com>

weekbeforenext committed:
DP-22331: Updating the mayflower-artifacts package

Commit:
DP-22959: Fixing the org_page migration logic for events and news (#1004)

Javi-er committed:
DP-22961 removing hotfix that's not longer relevant. (#1008)

Commit:
DP-22747: Run Backstop on flexible org branch (followup) (#1009)

* DP-22747: Fixing the Related locations section for Location pages with new org_page sections

* DP-22747: Refactoring logic to build the Roles section of a Person page

* DP-22747: Updating the changelog

* DP-22747: Fixing PHP standards errors

* DP-22747: Fixing the OrgPageMetadataTest PHPUnit test

* DP-22747: Fixing the OrgPageMetadataTest PHPUnit test

weekbeforenext committed:
Merge branch 'develop' into DP-22331_flex-orgs

Commit:
DP-22960: Org subnavigation fixes for feature branch (#1019)

* DP-22960: Refactoring the 'Find a service or task' section of the org_page 'I want to...' SubNav.

* DP-22960: Refactoring the 'Learn more about' section of the org_page SubNav 'I want to...'

* DP-22960: Cleaning up the org_page extra_node_all_actions field value

* DP-22960: Adding a changelog file

* DP-22960: Updating and adding more extra fields for org pages, using data within sections

* DP-22960: Fixing PHP coding standards errors

clairesunstudio committed:
merge in develop

clairesunstudio committed:
link to mayflower integration branch

Commit:
DP-22960: Org subnavigation fixes for feature branch (#1031)

* DP-22960: Refactoring the 'Find a service or task' section of the org_page 'I want to...' SubNav.

* DP-22960: Refactoring the 'Learn more about' section of the org_page SubNav 'I want to...'

* DP-22960: Cleaning up the org_page extra_node_all_actions field value

* DP-22960: Adding a changelog file

* DP-22960: Updating and adding more extra fields for org pages, using data within sections

* DP-22960: Fixing PHP coding standards errors

* DP-22960: Creating an reusable function for ensuring links to nodes have a correct title

* DP-22960: Fixing PHP coding standards issues

weekbeforenext committed:
DP-22331: Fixing missing composer package from develop merge

Commit:
DP-23023: Reorder paragraphs for org section (#1033)

* DP-23023: Alphabetizing the paragraph options for the Organization Section paragraph Content field

* DP-23023: Adding a changelog file

Commit:
DP-22960: Org subnavigation fixes for feature branch (followup config) (#1034)

* DP-22960: Refactoring the 'Find a service or task' section of the org_page 'I want to...' SubNav.

* DP-22960: Refactoring the 'Learn more about' section of the org_page SubNav 'I want to...'

* DP-22960: Cleaning up the org_page extra_node_all_actions field value

* DP-22960: Adding a changelog file

* DP-22960: Updating and adding more extra fields for org pages, using data within sections

* DP-22960: Fixing PHP coding standards errors

* DP-22960: Creating an reusable function for ensuring links to nodes have a correct title

* DP-22960: Fixing PHP coding standards issues

* DP-22960: Updating the org_page display configuration to include new extra fields

weekbeforenext committed:
Merge branch 'develop' into DP-22331_flex-orgs

Commit:
DP-23040: Investigate 500 error when editing orgs (#1037)

* DP-23040: Adding a line to the local services file to workaround the 500 error on migrated org pages

* DP-23040: Adding a changelog file

* DP-23040: Commenting out the parameters setting

Javi-er committed:
DP-22981 Adjust the org long form component to support sub grouped elements (#1041)

* DP-22981 Adjust the org long form component to support sub grouped elements.

* Create DP-22981.yml

* cleanup

Javi-er committed:
DP-22982 Iframe, Caspio and Tableau components on org page not rendering correctly (#1047)

* wrapping Caspio, Tableau and Iframes.

* formatting and clean up.

Javi-er committed:
DP-22983 Adding new variable for wrapping call out links in some situations. (#1046)

* Adding new variable for wrapping call out links in some situations.

* Update mass_theme.theme

* adjusting callout links paragraph.

* Update paragraph--what-would-you-like-to-do.html.twig

weekbeforenext committed:
Merge branch 'develop' into DP-22331_flex-orgs

weekbeforenext committed:
DP-22331: Updating the mayflower dev package

Javi-er committed:
removing unwanted separator printing on paragraph content.

Commit:
DP-23084: A11y don't render empty link tag for org key message with no link (#1048)

* DP-22767: Security updates (#985)

* Bump path-parse from 1.0.6 to 1.0.7 in /cloudflare

Bumps [path-parse](https://github.com/jbgutierrez/path-parse) from 1.0.6 to 1.0.7.
- [Release notes](https://github.com/jbgutierrez/path-parse/releases)
- [Commits](https://github.com/jbgutierrez/path-parse/commits/v1.0.7)

---
updated-dependencies:
- dependency-name: path-parse
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

* DP-22767: Mass.gov security updates

* DP-22767: Mass.gov security updates

* Updated Drupal core and Entity Embed - DP-22767

* Updated changelog - DP-22767

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Tom Fleming <tom@lastcallmedia.com>

* DP-23084: Updating the Feature Message paragraph template to conditionally show the callout link

Co-authored-by: Arthur Baghdasaryan <62405127+arthurbaghdas@users.noreply.github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Tom Fleming <tom@lastcallmedia.com>

Javi-er committed:
making images to be wrapped up on organization sections.

Javi-er committed:
Merge branch 'DP-22331_flex-orgs' of github.com:massgov/openmass into DP-22331_flex-orgs

Commit:
changing help text for featured services paragraph

weekbeforenext committed:
Merge branch 'develop' into DP-22331_flex-orgs

weekbeforenext committed:
Merge branch 'develop' into DP-22331_flex-orgs

weekbeforenext committed:
DP-22981: Updating the mayflower dev version

clairesunstudio committed:
test only org pages

Javi-er committed:
DP-22982 Iframe, Caspio and Tableau components on org page not rendering correctly (#1055)

* Adjusting caspio, tableau amd iframes.

* Update mass_theme.theme

* cleaning up

weekbeforenext committed:
Merge branch 'develop' into DP-22331_flex-orgs

clairesunstudio committed:
Revert "test only org pages"

This reverts commit 55166cadc4ddc987b68b8760b43823846608d4f9.

clairesunstudio committed:
Merge branch 'DP-22331_flex-orgs' of https://github.com/massgov/openmass into DP-22331_flex-orgs

Javi-er committed:
DP-22982 iframe, caspio, tableau (and images)  (#1058)

* Adjusting caspio, tableau amd iframes.

* Update mass_theme.theme

* cleaning up

* DP-22982 Added no sidebar class to mimic info details, cleaning up, improving comments.

* cleaning up

Javi-er committed:
DP-22982 Iframe, Caspio and Tableau components on org page not rendering correctly (#1059)

* Adjusting caspio, tableau amd iframes.

* Update mass_theme.theme

* cleaning up

* DP-22982 Added no sidebar class to mimic info details, cleaning up, improving comments.

* cleaning up

* adding image overrides for organizations.

weekbeforenext committed:
Merge branch 'develop' into DP-22331_flex-orgs

weekbeforenext committed:
DP-22982: Updating the mayflower version

Javi-er committed:
Dp 22982 rich text fix (#1063)

* Making rich text to be contained when standalone.

* Update docroot/themes/custom/mass_theme/overrides/css/contained.css

Co-authored-by: Minghua Sun <clairesunstudio@gmail.com>

* cleaning up

Co-authored-by: Minghua Sun <clairesunstudio@gmail.com>

Commit:
help text change

weekbeforenext committed:
DP-22331: Updating the mayflower version

Commit:
help text change

weekbeforenext committed:
DP-22331: Updating mayflower version

Commit:
removed backstop page that is no longer used

Commit:
DP-23093: Update org report on flexible org feature branch (#1065)

* DP-23093: Moving the Content Report: Organization Pages to its own view and updating the fields to show types of content in organization sections

* DP-23093: Adding the changelog file

* DP-23093: Fixing a PHP standards error

* DP-23093: Fixing a PHP standards error

* DP-23093: Fixing a PHP standards error

weekbeforenext committed:
DP-22331: Deleting an old changelog that has already deployed to production
